crates/runtime/src/datafusion/builder.rs

```diff
@@ -170,6 +173,7 @@ fn get_analyzer_rules() -> Vec<Arc<dyn AnalyzerRule + Send + Sync>> {
     vec![
         Arc::new(InlineTableScan::new()),
         Arc::new(ExpandWildcardRule::new()),
+        Arc::new(SpiceUDFsOverride::new()),
```

Comment:
New rule is added before FederationAnalyzerRule so it is universal and could be used with and without federation (same logic w/o additional implementation specifically for FederationAnalyzerRule)